app-games/src/main/java/com/aptoide/android/aptoidegames/toolbar/AppGamesTopBar.kt

```diff
@@ -78,14 +63,15 @@ private fun TopBar(
     )
     Text(
       text = title,
+      color = agWhite,
       modifier = Modifier
         .fillMaxWidth()
         .weight(1f)
-        .padding(start = 16.dp, top = 16.dp, bottom = 16.dp),
+        .padding(),
       textAlign = TextAlign.Center,
       overflow = TextOverflow.Ellipsis,
       maxLines = 1,
-      style = AppTheme.typography.headlineTitleText
+      style = AppTheme.typography.title
     )
     Spacer(Modifier.width(64.dp))
```

Comment:
I think this layout could be easier to understand by moving the horizontal padding to the parent row, and keep this spacer with the same size of the image